infrastructure/google-cloud-run/start-nginx.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+export EXISTING_VARS=$(printenv | awk -F= '{print $1}' | sed 's/^/\$/g' | paste -sd,);
+
+cat config/default-env.json | envsubst $EXISTING_VARS | tee config/default.json
```

Comment:
If you are selecting all var, isn't that the same as running `envsubst` with no parameters?